src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -211,17 +212,21 @@ export class QuickBar extends LitElement {
           domainToName(this.hass.localize, domain)
         ),
       icon: domainIcon(domain),
+      score: 0,
```

Comment:
Why add a score here?

Reply:
Just made the sort code a little cleaner, but it is kind of weird. I'll remove

Reply:
Done